end-to-end-tests/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
The tests run in node and we use a very new version of node so this can be a much newer `target` (and i'd like for it to be)

Reply:
ES6 seems to be a safe choice, with most modern browsers supporting it. Do you have a preference?

Reply:
I don't see much reason to not use ECMAScript 2022

Reply:
The reason I'm hesitant on using the latest version is due to some browser support not being available to some of the newest features. I'm not sure how much we care about, "Class static initialization blocks" but it's global support is kinda low (73.85%) and no support for Safari. Although maybe none of users use Safari/older browser versions and this isn't a feature we will use - so maybe this something we really need to care about.

Reply:
> When I'm speaking about our users who use Safari/older browsers I'm speaking in terms of other developers who are running our cypress tests and using these browsers to do the testing in - this may be a very rare case, but it may have an impact on them.

Let's worry about that level of compatibility in `oscal-react-library` but I am less concerned about it here. Let's go with ES2021 or ES2022 and we can drop that if we see problems.

I don't think we're using any features where it really matters in the tests. But it'll be good to use something relatively recent.